src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimizeBatched.cpp

```diff
@@ -750,18 +746,17 @@ bool QMCFixedSampleLinearOptimizeBatched::processOptXML(xmlNodePtr opt_xml,
 
   vmcEngine->enable_sample_collection();
 
-  // Code to check and set crowds take from QMCDriverNew::adjustGlobalWalkerCount
-  checkNumCrowdsLTNumThreads(opt_num_crowds_);
-  if (opt_num_crowds_ == 0)
-    opt_num_crowds_ = Concurrency::maxCapacity<>();
 
-  app_log() << " Number of crowds for optimizer: " << opt_num_crowds_ << std::endl;
+  QMCDriverNew::AdjustedWalkerCounts awc =
```

Comment:
If you prefer not to see the warning again. We can add an option to suppress warnings in adjustGlobalWalkerCount

Reply:
You expressed concerns of printing duplicated warning. How would you like to handle this?
I'm actually fine with printing warning twice. First, the occurrence is low. If users decide to live with the warning, they can live with twice warning. Second, if we can reuse walker objects, then warning is also printed once only.
So I don't really feel this is critical. let me know your choice to add a warning suppression flag or leave it.

Reply:
Printing the warning twice is fine for now.  If it becomes an issue later, we can add the flag.